<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickInstallItemClass</class>
 <widget class="QPushButton" name="QuickInstallItemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>99</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>172</width>
    <height>99</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>172</width>
    <height>99</height>
   </size>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="quick_icon">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>13</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="quick_name">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>16</y>
     <width>70</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>高德地图</string>
   </property>
  </widget>
  <widget class="QLabel" name="quick_detail">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>72</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>最有效的地图</string>
   </property>
  </widget>
  <widget class="QLabel" name="quick_rec">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>17</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quick_state">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>37</y>
     <width>58</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>安装</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../UmInstallerAssistant.qrc"/>
 </resources>
 <connections/>
</ui>
